firebase-vscode/src/workflow.ts

```diff
@@ -145,26 +145,32 @@ export function setupWorkflow(
 ) {
   extensionContext = context;
 
-  // Get user-defined VSCode settings.
-  const workspaceConfig = workspace.getConfiguration(
-    'firebase',
-    vscode.workspace.workspaceFolders[0].uri
-  );
-  const shouldDebug: boolean = workspaceConfig.get('debug');
+  var shouldDebug: boolean;
+  if (vscode.workspace.workspaceFolders) {
```

Comment:
As I mentioned in the chat, this may get the user past init without errors but they will run into errors when the extension tries to find the root folder - we should try to figure out if we should allow people to use the extension at all without providing a root folder. I think we should probably prompt them to provide a path or open a folder first

Reply:
This is a little fragile but works in the near-term - the extension loads and lets you select a project. I agree we should have some explicit checking in the future